discovery/disc_menu.py

```diff
@@ -73,42 +72,48 @@ def disc_menu():
         # --------------------------------------------------- FINVIZ ---------------------------------------------------
         elif ns_known_args.cmd == 'map':
             finviz_api.map_sp500(l_args)
-        
+
         # ------------------------------------------------ ALPHA VANTAGE -----------------------------------------------
         elif ns_known_args.cmd == 'sectors':
             alpha_vantage_api.sectors(l_args)
-        
-        # ------------------------------------------------ YAHOO FINANCE ------------------------------------------------
+
+        # ------------------------------------------------ YAHOO FINANCE
+        # ------------------------------------------------
```

Comment:
This doesn't look great. Could you make sure that it still remained in one line?